pkgs/development/python-modules/thefuzz/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildPythonPackage, fetchPypi, python-Levenshtein, pycodestyle, hypothesis, pytest }:
+
+buildPythonPackage rec {
+  pname = "thefuzz";
+  version = "0.19.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-b3Em2y8silQhKwXjp0DkX0KRxJfXXSB1Fyj2Nbt0qj0=";
+  };
+
+  propagatedBuildInputs = [ python-Levenshtein ];
+  checkInputs = [ pycodestyle hypothesis pytest ];
```

Patch out the `pycodestyle` requirement as we don't want to lint code. Please check if it's possible to use 

```suggestion
  checkInputs = [
    hypothesis
    pytestCheckHook
  ];
```